### PR TITLE
Added fc22 to dist_list.txt

### PR DESCRIPTION
**Pull request by dkliban**

### CODE REVIEW

**Comment by ipanova** [comment]
ok test

**Comment by rbarlow** [comment]
ok test

File: dist_list.txt [hunk 1]
```diff
@@ -1 +1 @@
-el6 el7 fc20 fc21
\ No newline at end of file
+el6 el7 fc20 fc21 fc22
```
Comment (rbarlow) [inline dist_list.txt]: I think we should also drop support for F20, since it is only supported for two more days.

**Comment by rbarlow** [comment]
The tests fail on the 1.0-* branches due to some nonsense that isn't worth fixing IMO. So don't worry about that. LGTM!

**Pull request merged**